package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -165,3 +176,212 @@ def _read_first_frame(self):
+        self.file.write('Atoms\n')
+        self.file.write('\n')
+
+        try:
+            charges = atoms.charges
+        except (NoDataError, AttributeError):
+            has_charges = False
+        else:
+            has_charges = True
+
+        indices = atoms.indices + 1
+        types = atoms.types.astype(np.int32)
+
+        if self.convert_units:
+            coordinates = self.convert_pos_to_native(atoms.positions, inplace=False)
+
+        if has_charges:
+            for index, atype, charge, coords in zip(indices, types, charges,
+                    coordinates):
+                self.file.write('%d 0 %d %f %f %f %f\n'%(index, atype,
+                        charge, coords[0], coords[1], coords[2]))
+        else:
+            for index, atype, coords in zip(indices, types, coordinates):
+                self.file.write('%d 0 %d %f %f %f\n'%(index, atype,\
+                        coords[0], coords[1], coords[2]))
+
+    def _write_velocities(self, atoms):
+        self.file.write('\n')
+        self.file.write('Velocities\n')
+        self.file.write('\n')
+        indices = atoms.indices + 1
+        velocities = self.convert_velocities_to_native(atoms.velocities, inplace=False)
+        for index, vel in zip(indices, velocities):
+            self.file.write('%d %f %f %f\n'%(index, vel[0], vel[1], vel[2]))
+
+    def _write_masses(self, atoms):
+        self.file.write('\n')
+        self.file.write('Masses\n')
+        self.file.write('\n')
+        mass_dict = {}
+        for atype in set(atoms.types.astype(np.int32)):
+            masses = set(atoms.select_atoms('type %s'%(atype)).masses)
+            mass_dict[atype] = masses.pop()
+            if masses:
+                raise ValueError('LAMMPS DATAWriter: to write data file, '+\
+                        'atoms with same type must have same mass')
+        for atype, mass in mass_dict.items():
+            self.file.write('%d %f\n'%(atype, mass))
+
+    def _write_bonds(self, bonds):
+        num = len(bonds[0])
+        self.file.write('\n')
+        self.file.write('%s\n'%btype_sections[bonds.btype])
+        self.file.write('\n')
+        for bond, i in zip(bonds, range(1, len(bonds)+1)):
+            self.file.write('%d %s '%(i, bond.type)+\
+                    ' '.join((bond.atoms.indices + 1).astype(str))+'\n')
+
+    def _write_dimensions(self, dimensions):
+        """
+            Convert dimensions to triclinic vectors, convert lengths to native
+            units and then write the dimensions section
+        """
+        if self.convert_units:
+            triv = self.convert_pos_to_native(mdamath.triclinic_vectors(\
+                    dimensions),inplace=False)
+        self.file.write('\n')
+        self.file.write('%f %f xlo xhi\n'%(0., triv[0][0]))
+        self.file.write('%f %f ylo yhi\n'%(0., triv[1][1]))
+        self.file.write('%f %f zlo zhi\n'%(0., triv[2][2]))
+        if any(np.array([triv[1][0], triv[2][0], triv[2][1]]) != 0):
```

Comment:
So anything that isn't 0 is True, so this could boil down to `if any([triv[1][0], triv[2][0], triv[2][1]]):`, no need to make the array either